.goreleaser.yml

```diff
@@ -0,0 +1,110 @@
+project_name: erigon
+
+release:
+  disable: false
+  draft: true
+  prerelease: auto
+
+builds:
+  - id: darwin-amd64
+    main: ./cmd/erigon
+    binary: erigon
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    env:
+      - CC=o64-clang
+      - CXX=o64-clang++
+    tags:
+      - netgo
+    ldflags:
+      -s -w
+
+  - id: darwin-arm64
+    main: ./cmd/erigon
+    binary: erigon
+    goos:
+      - darwin
+    goarch:
+      - arm64
+    env:
+      - CC=oa64-clang
+      - CXX=oa64-clang++
+    tags:
+      - netgo
+    ldflags:
+      -s -w
+
+  - id: linux-amd64
+    main: ./cmd/erigon
+    binary: erigon
+    goos:
+      - linux
+    goarch:
+      - amd64
+    env:
+      - CC=gcc
+      - CXX=g++
+    tags:
+      - netgo
+    ldflags:
+      # We need to build a static binary because we are building in a glibc based system and running in a musl container
+      -s -w -extldflags "-static"
+
+  - id: linux-arm64
+    main: ./cmd/erigon
+    binary: erigon
+    goos:
+      - linux
+    goarch:
+      - arm64
+    env:
+      - CC=aarch64-linux-gnu-gcc
+      - CXX=aarch64-linux-gnu-g++
+    tags:
+      - netgo
+    ldflags:
+      # We need to build a static binary because we are building in a glibc based system and running in a musl container
+      -s -w -extldflags "-static"
+
+snapshot:
+  name_template: "{{ .Tag }}.next"
+
+dockers:
+  - image_templates:
+      - 0xpolygon/{{ .ProjectName }}:{{ .Version }}-amd64
```

Comment:
- polygon is the fork of Erigon? why it has own binary?

Reply:
Ah sorry, missed this when duplicating the functionality. Will fix.